packages/fast-element/src/dom.ts

```diff
@@ -4,7 +4,35 @@ const markerClass = `fast-${Math.random()
     .substring(7)}`;
 const updateQueue = [] as Callable[];
 
-export const DOM = {
+type TrustedTypesPolicy = { createHTML(html: string): string };
+
+// Tiny API-only polyfill for trustedTypes
+if (globalThis.trustedTypes === void 0) {
+    globalThis.trustedTypes = { createPolicy: (name, rules) => rules };
+}
+
+const fastHTMLPolicy: TrustedTypesPolicy = globalThis.trustedTypes.createPolicy(
+    "fast-html",
+    {
+        createHTML: html => html,
+    }
+);
+
+let htmlPolicy: TrustedTypesPolicy = fastHTMLPolicy;
+
+export const DOM = Object.freeze({
```

Comment:
The `DOM` object is nor frozen so that it's `createHTML` method cannot be overwritten by attackers.

Reply:
nice!